app/src/main/res/values/strings.xml

```diff
@@ -42,7 +42,7 @@
     <string name="action_enter_zipcode">enter zipcode</string>
     <string name="text_minimum_temp">61</string>
     <string name="text_maximum_temp">78</string>
-    <string name="text_temperature">70</string>
+    <string name="text_temperature">70°</string>
```

Comment:
What is this for?

Reply:
That was for me to check because the width of the temperature component was limited and was not showing the degree sign.